### DISCUSSION TITLE
Is there any way to access the code for implementing chat commands?

### DISCUSSION THREAD

**Discussion by rishivinod7** [mozilla/hubs#3195] [General Questions & Discussion] [answered]
Hi, I would like to see if its possible to make command that can modify some audio params. If anyone has any advice or tips that'd be great!

**Comment by robinkwilson** [comment, marked as answer]
Recently this was merged into the master branch (not live on Hubs Cloud yet): https://github.com/mozilla/hubs/pull/2824

This PR implemented a chat command to configure audio normalization, I think this is a good reference to point you in the right direction.

Also I'd recommend joining the hubs community discord server to search past discord discussions related to audio or collaborate with community members.

**Reply by rishivinod7** [reply]
Thanks I'll look into that!
